pyanaconda/kickstart.py

```diff
@@ -18,69 +18,64 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 
-from pyanaconda.errors import ScriptError, errorHandler
-from pyanaconda.threading import threadMgr
-from blivet.deviceaction import ActionCreateFormat, ActionResizeDevice, ActionResizeFormat
-from blivet.devices import LUKSDevice
-from blivet.devices.lvm import LVMVolumeGroupDevice, LVMCacheRequest, LVMLogicalVolumeDevice
-from blivet.devicelibs.lvm import LVM_PE_SIZE, KNOWN_THPOOL_PROFILES
-from blivet.devicelibs.crypto import MIN_CREATE_ENTROPY
-from blivet.formats import get_format
-from blivet.formats.fs import XFS
-from blivet.partitioning import do_partitioning
-from blivet.partitioning import grow_lvm
-from blivet.errors import PartitioningError, StorageError, BTRFSValueError
-from blivet.size import Size, KiB
-from blivet import udev
-from blivet import autopart
-from blivet.platform import platform
-import blivet.iscsi
-import blivet.fcoe
-import blivet.zfcp
-import blivet.arch
-
 import glob
-from pyanaconda import iutil
 import os
 import os.path
-import tempfile
-from pyanaconda.flags import flags, can_touch_runtime_system
-from pyanaconda.constants import ADDON_PATHS, IPMI_ABORTED, TEXT_ONLY_TARGET, GRAPHICAL_TARGET, THREAD_STORAGE
-import shlex
 import requests
+import shlex
 import sys
-import pykickstart.commands as commands
-import warnings
+import tempfile
 import time
-from pyanaconda import keyboard
-from pyanaconda import ntp
-from pyanaconda import timezone
-from pyanaconda.timezone import NTP_PACKAGE, NTP_SERVICE
-from pyanaconda import localization
-from pyanaconda import network
-from pyanaconda import nm
-from pyanaconda.simpleconfig import SimpleConfigFile
-from pyanaconda.users import getPassAlgo
+import warnings
+
+import blivet.arch
+import blivet.fcoe
+import blivet.iscsi
+import blivet.zfcp
+
+import pykickstart.commands as commands
```

Comment:
It is here intentionally because it is an `import` and the other `pykickstart` are from. This is looking better for an eye to me.